### ISSUE TITLE
Get the voice of a single person

### ISSUE THREAD

**Issue by Tockra** [ReSpeak/tsclientlib#26]
Hi Guys,

I wanted to ask if this crate allows to process the voice of the persons in a channel.
So if the bot is in a channel with 3 persons, than he should get the voice of each person separately without mixing them.
Is that possible?

T

**Comment by Flakebi** [comment]
Yes, you get the voice unmixed for every person. If you want it mixed, you would need to do that explicitly, for example with the [`AudioHandler`](https://docs.rs/tsclientlib/0.2.0/tsclientlib/audio/struct.AudioHandler.html).

The example code here receives audio for every client and hands them off to another thread (that will then mix it together).
https://github.com/ReSpeak/tsclientlib/blob/f8a05048c5d4f6b89aeb5f01130c2adacb395bf2/tsclientlib/examples/audio.rs#L89-L102
